### ISSUE TITLE
Might seed buff not displaying

### ISSUE THREAD

**Issue by lokiregnarok** [sir-wilhelm/SmartHunter#30]
As the title says. All other buffs are displaying correctly. Cannot see might seed buff timer

**Comment by sir-wilhelm** [comment]
Can you make sure you are running the latest version/copy your logs for me?

1.5.5.0 is working for me:
![image](https://user-images.githubusercontent.com/3122458/79480868-4ecbe100-7fd4-11ea-96eb-93d3674e744c.png)
```text
[2020-04-16 16:16:17] Started 1.5.5.0
[2020-04-16 16:16:18] Config.json loaded
[2020-04-16 16:16:18] en-US.json loaded
[2020-04-16 16:16:18] MonsterData.json loaded
[2020-04-16 16:16:18] PlayerData.json loaded
[2020-04-16 16:16:18] Memory.json saved
[2020-04-16 16:16:18] Default.xaml loaded
[2020-04-16 16:16:19] State Machine: None (61 ms) > DeleteOldFiles
[2020-04-16 16:16:19] State Machine: DeleteOldFiles (49 ms) > CheckingForUpdates
[2020-04-16 16:16:19] Searching for updates...
[2020-04-16 16:16:19] This can be disabled by setting ["AutomaticallyCheckAndDownloadUpdates": true,] in Config.json.
[2020-04-16 16:16:20] No updates found.
[2020-04-16 16:16:20] State Machine: CheckingForUpdates (468 ms) > WaitingForProcess
[2020-04-16 16:16:20] State Machine: WaitingForProcess (70 ms) > ProcessFound
[2020-04-16 16:16:20] State Machine: ProcessFound (49 ms) > FastPatternScanning
[2020-04-16 16:16:20] State Machine: FastPatternScanning (52 ms) > PatternScanning
[2020-04-16 16:16:20] Base: 0x140000000, End: 0x169233000, Size: 0x29233000
[2020-04-16 16:16:20] Found 'MonsterPattern' at address 0x14023A499
[2020-04-16 16:16:20] Found 'SelectedMonsterPattern' at address 0x1419C5DD2
[2020-04-16 16:16:20] Found 'PlayerDamagePattern' at address 0x14198D747
[2020-04-16 16:16:20] Found 'CurrentWeaponPattern' at address 0x141F294ED
[2020-04-16 16:16:21] Found 'CurrentPlayerNamePattern' at address 0x1474F2329
[2020-04-16 16:16:21] Found 'LobbyStatusPattern' at address 0x14752ADC4
[2020-04-16 16:16:21] Found 'PlayerNamePattern' at address 0x14894FC97
[2020-04-16 16:16:25] Found 'PlayerBuffPattern' at address 0x164E32B21
[2020-04-16 16:16:25] State Machine: PatternScanning (5201 ms) > Working
[2020-04-16 16:16:25] Match Range: 14023A499 - 164E32B21
```

**Comment by sir-wilhelm** [comment]
I just released 1.5.6.0, but it did not change anything for Might seed and it should be working in that version too.

**Comment by lokiregnarok** [comment]
to update on that, it seems to be working anywhere else. just not in guiding lands.
